rtdata/languages/default

```diff
@@ -3125,7 +3127,6 @@ TP_LOCALLAB_MERNIN;Screen
 TP_LOCALLAB_MERONE;Normal
 TP_LOCALLAB_MERSAT;Saturation
 TP_LOCALLAB_MERSEV;Soft Light (legacy)
-TP_LOCALLAB_MERSEV0;Soft Light Illusion
```

Comment:
Probably should not be deleted

Reply:
I confirm 
TP_LOCALLAB_MERSEV0;Soft Light Illusion
Must not be deleted

Jacques

Reply:
Restored.